src/foam/nanos/crunch/ui/UCJView.js

```diff
@@ -127,23 +127,21 @@ foam.CLASS({
       });
       var sequence = undefined;
       if ( this.isSettingCapabilities ) {
-        x = x.createSubContext({
-          capabilities: this.capabilitiesList
-        });
-        sequence = this.Sequence.create(null, x)
-        .add(this.ConfigureFlowAgent, { popupMode: false })
-        .add(this.WAOSettingAgent)
-        .add(this.DeveloperModeAgent)
-        .add(this.CreateWizardletsAgent)
-        .add(this.LoadWizardletsAgent)
-        .add(this.CreateControllerAgent)
-        .add(this.StepWizardAgent)
-        .add(this.DetachAgent)
-        .add(this.SpinnerAgent)
-        .add(this.SaveAllAgent)
-        .add(this.SubmitAgent)
-        .add(this.DetachSpinnerAgent)
-        .add(this.CapabilityStoreAgent);
+        sequence = this.crunchController.createWizardSequence(this.capabilitiesList, x);
```

Comment:
Was this tested? this flow is for the user 'all ucj' tab...if it does work we should probably figure out if we have a redundant property.

this.crunchController.createWizardSequence(this.capabilitiesList, x); ... this will set this.capabilitiesList as the rootCapabilityId ... which isn't what we want ... we want capabilities: this.capabilitiesList in the context